task-4/task-4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace task_4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Hello, user!");
+            FullEquationCalculator();
+            Console.WriteLine("/// type to exit");
+            Console.ReadKey();
+        }
+
+        static void FullEquationCalculator() 
+        {
+            double answer = smallCalc(3, 5) * smallCalc(3, 7) * smallCalc(5, 5);
+            Console.WriteLine($"Your answer is {Math.Round(answer, 2)}");
+        }
+
+        static double smallCalc(double arg1, double arg2)
```

Comment:
В C# принято называть методы с большой буквы. Кроме того, лучше если название метода начинается с глагола. Имена параметров метода ни о чем не говорят, так что их можно было выбрать и покороче, например,
```suggestion
        static double CalculateSmallExpression(double a, double b)
```